docusaurus/docs/operate/cheat_sheets/supplier_cheatsheet.md

```diff
@@ -37,6 +39,23 @@ streamline development and reduce friction for any new potential contributor.
   - [What Supplier queries are available?](#what-supplier-queries-are-available)
   - [How do I query for all existing onchain Suppliers?](#how-do-i-query-for-all-existing-onchain-suppliers)
 
+## System Requirements
+
+CPU: 2 Cores/4 Threads
```

Comment:
Open to push back here but I do want to suggest an alternative.

1. Instead of text, use tables
2. Consolidate all the hardware requirements in one place
3. Link to it from here

See this page as a reference: https://dev.poktroll.com/operate/configs/hardware_requirements